src/lib/api/org/index.ts

```diff
@@ -275,3 +277,108 @@ function convertToSnykProject(projectData: v3ProjectData[]) : SnykProject[] {
   return projects
 }
 
+export async function listTargets(
+  requestManager: requestsManager,
+  orgId: string,
+  config?: {
```

Comment:
if the "config" itself is optional, wouldn't that make the entries inside automatically optional as well?

Reply:
no, if I don't put the optional in the config then I need all of them to be defined if config is set but I don't I can have only one of the option.